.drone.yml

```diff
@@ -42,23 +42,33 @@ pipeline:
       - cd /drone/server; php ./occ a:l
       - cd /drone/server; php ./occ a:e files_primary_s3
       - cd /drone/server; php ./occ a:l
-      - cp /drone/server/apps/files_primary_s3/tests/drone/scality.config.php /drone/server/config
+    when:
+      local: false
```

Comment:
Now `drone exec` won't work anymore...

Reply:
which is intended - I assume that the app lives already in an owncloud instance

Reply:
But the server is not available within the drone context. You can drop this to keep it working with drone exec

Reply:
Hint: only the workspace path gets mounted into the drone context